src/pages/home/report/ReportAttachments.tsx

```diff
@@ -1,43 +1,47 @@
-import PropTypes from 'prop-types';
+import {StackScreenProps} from '@react-navigation/stack';
 import React, {useCallback} from 'react';
-import _ from 'underscore';
 import AttachmentModal from '@components/AttachmentModal';
 import ComposerFocusManager from '@libs/ComposerFocusManager';
 import Navigation from '@libs/Navigation/Navigation';
+import {AuthScreensParamList} from '@libs/Navigation/types';
 import * as ReportUtils from '@libs/ReportUtils';
 import ROUTES from '@src/ROUTES';
+import SCREENS from '@src/SCREENS';
 
-const propTypes = {
-    /** Navigation route context info provided by react navigation */
-    route: PropTypes.shape({
-        /** Route specific parameters used on this screen */
-        params: PropTypes.shape({
-            /** The report ID which the attachment is associated with */
-            reportID: PropTypes.string.isRequired,
-            /** The uri encoded source of the attachment */
-            source: PropTypes.string.isRequired,
-        }).isRequired,
-    }).isRequired,
+type File = {
```

Comment:
I just wonder if this type shouldn't be global

Reply:
I think you are right , I remember that in other Attachments files we should use the same type , I will move it to other place , thanks!

Reply:
There is a type for File defined globally, but it has a different structure:
```ts
    interface File {
        source: string;
        uri: string;
    }
```